src/index.html

```diff
@@ -21,18 +21,7 @@
             </form>
             
             <ul class="todo-items">
-                <!-- loads dynamically -->
-                <!-- <li class="flex-item" id="${item.index}">
-                    <div class="list-item">
-                        <input type="checkbox" class="checkbox">
-                        <input type="text" class="desc" value="something">
-                    </div>
-                    <i class="fa-solid fa-ellipsis-vertical" id="drag"></i>
-                    <i class="fa-regular fa-trash-can hidden" id="delete"></i>
-                </li> -->
-
-                
-                
```

Comment:
- [ ] Kindly remove any commented code in your HTML file, since it is not good practice to have them.